vlcplayer/__init__.py

```diff
@@ -55,6 +55,12 @@ def previous(self):
         if self.is_playing():
             self.list_player.previous()
 
+    def restart(self):
+        if self.is_playing():
+            self.list_player.previous()
+            time.sleep(0.01)
+            self.list_player.next()
```

Comment:
Does this work with the first song in a play list, too?

Reply:
Yes, I tested it for a single song mrl and for youtube(playytb) as well. Testing and suggestions for a better approach is welcome.